db/migrations/0016_auto_apply_pending_demotion.sql

```
@@ -0,0 +1,15 @@
+ALTER TABLE "triage_config" ADD COLUMN "auto_apply_promotions" boolean DEFAULT false NOT NULL;--> statement-breakpoint
+CREATE TABLE "pending_demotion" (
+	"id" bigserial PRIMARY KEY NOT NULL,
+	"owner_auth_user_id" text NOT NULL,
+	"gmail_message_id" text NOT NULL,
+	"verdict_id" bigint NOT NULL,
+	"confirmed_at" timestamp with time zone,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "pending_demotion" ADD CONSTRAINT "pending_demotion_owner_auth_user_id_owner_binding_auth_user_id_fk" FOREIGN KEY ("owner_auth_user_id") REFERENCES "public"."owner_binding"("auth_user_id") ON DELETE no action ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)